snuba/datasets/storages/spans_v2.py

```diff
@@ -0,0 +1,63 @@
+from snuba.clickhouse.columns import UUID, ColumnSet, DateTime, Float
+from snuba.clickhouse.columns import SchemaModifiers as Modifiers
+from snuba.clickhouse.columns import String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.schemas.tables import WritableTableSchema
+from snuba.datasets.spans_processor import SpansMessageProcessorV2
+from snuba.datasets.storage import WritableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.table_storage import build_kafka_stream_loader_from_settings
+from snuba.query.processors.type_converters.hexint_column_processor import (
+    HexIntColumnProcessor,
+)
+from snuba.utils.streams.topics import Topic
+from snuba.web.split import TimeSplitQueryStrategy
+
+columns = ColumnSet(
+    [
+        ("project_id", UInt(64)),
+        ("transaction_id", UUID()),
+        ("trace_id", UUID()),
+        ("transaction_span_id", UInt(64)),
+        ("span_id", UInt(64)),
+        ("parent_span_id", UInt(64, Modifiers(nullable=True))),
+        ("transaction_name", String()),
+        ("span_group_id", UUID()),
+        ("op", String()),
+        ("status", UInt(8)),
+        # transaction start/finish times
+        ("start_ts", DateTime()),
+        ("start_ns", UInt(32)),
+        ("finish_ts", DateTime()),
+        ("finish_ns", UInt(32)),
```

Comment:
We require the transaction's start/finish times here as it needs to be in the order by clause to help with the merge join.